app/controllers/repository_statistics_controller.rb

```diff
@@ -0,0 +1,56 @@
+class RepositoryStatisticsController < ApplicationController
+
+  CHART_COMPONENTS = ["Report", "Image", "Thesis", "Journal Article (Published)"]
```

Comment:
Sorry - the intention was not to pin these specific values, but to show the top four, whatever they may be. So the facet query should determine which four (+ "Other") get displayed.